### PR TITLE
Upgrade to PHP-DI 7

### PR DESCRIPTION
**Pull request by clsmedia**
Updated composer.json to use php-di 7 and tests to use Laminas\Diactoros instead of Zend.
I hope thats enough :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     },
     "require": {
         "php": "^7.1 || ^8.0",
-        "php-di/php-di": "^6.0.0",
+        "php-di/php-di": "^7.0.0",
```
Comment (mnapoli) [inline composer.json]: I'm thinking we could support `^6.0|^7.0` if there are no downsides?

**Review by mnapoli — COMMENTED** [review]
Thank you!

File: composer.json (hunk 1 above)
Comment (clsmedia) [reply composer.json]: I didn't find any. Should be fine!

Comment (mnapoli) [reply composer.json]: 👍 I'll wait for that change before merging then.

Comment (clsmedia) [reply composer.json]: Done

**Comment by mnapoli** [comment]
Thanks!

**Pull request merged**